packages/x-codemod/src/v6.0.0/pickers/replace-tabs-props/actual.spec.js

```diff
@@ -0,0 +1,19 @@
+<DateTimePicker
+  componentsProps={{
+    tabs: {
+      dateRangeIcon: <LightModeIcon />,
+    },
+  }}
+/>;
+
+<DateTimePicker
+  componentsProps={{
+    tabs: {
+      dateRangeIcon: <LightModeIcon />,
+    },
+  }}
+>
+  ChildrenString
+</DateTimePicker>;
```

Comment:
I struggled to make it work with self-closing items, so I preferred to add a tests for both self closing and with children. But effectively, the one with children could be removed since it does not exist 🤭